wp/lib/bap_wp/src/output.ml

```diff
@@ -60,6 +60,48 @@ let format_model (model : Model.model) (env1 : Env.t) (_env2 : Env.t) : string =
   Format.flush_str_formatter ()
 
 
+type mem_model = {default : Z3.Expr.expr ; model : (Z3.Expr.expr * Z3.Expr.expr) list}
```

Comment:
Not super important, but we've been using `Constr.z3_expr` instead of `Z3.Expr.expr` and I think it would be good to be consistent.